Wikipedia/Code/WMFOnThisDayContentSource.m

```diff
@@ -130,7 +131,12 @@ - (void)loadContentForDate:(NSDate *)date inManagedObjectContext:(NSManagedObjec
 
                         NSArray *eventsSortedByScore = [onThisDayEvents sortedArrayUsingDescriptors:@[[NSSortDescriptor sortDescriptorWithKey:@"score" ascending:NO]]];
                         if (eventsSortedByScore.count > 0) {
-                            featuredEventIndex = ((WMFFeedOnThisDayEvent*)eventsSortedByScore.firstObject).index.integerValue;
+                            // Rotate through the 10 highest scoring events based on current year. Ensures unlikely to see a repeat for at least 10 years
```

Comment:
@joewalsh I used some of your comments from the phab ticket for future reference.